recipes/mp-units/2.0.0/conanfile.py

```diff
@@ -59,11 +59,11 @@ def _use_range_v3(self):
         return "clang" in compiler and compiler.libcxx == "libc++" and version < 14
 
     def requirements(self):
-        self.requires("gsl-lite/0.40.0")
+        self.requires("gsl-lite/0.41.0")
         if self._use_libfmt:
-            self.requires("fmt/10.1.0")
+            self.requires("fmt/10.2.0")
```

Comment:
Did you check that 10.2.0 works properly? We are reusing some fmt internal implementation details. If those changed code might be broken.

Reply:
@mpusz 
Oh, I missed your PR.
I closed it.